static/app/components/events/interfaces/crashContent/stackTrace/content.tsx

```diff
@@ -172,240 +180,188 @@ class Content extends Component<Props, State> {
     return penultimateFrame.inApp && !lastFrame.inApp;
   }
 
-  isRepeatedFrame(frame: Frame, nextFrame?: Frame): boolean {
-    if (!nextFrame) {
-      return false;
-    }
-    return (
-      frame.lineNo === nextFrame.lineNo &&
-      frame.instructionAddr === nextFrame.instructionAddr &&
-      frame.package === nextFrame.package &&
-      frame.module === nextFrame.module &&
-      frame.function === nextFrame.function
-    );
+  function handleToggleAddresses(mouseEvent: React.MouseEvent<SVGElement>) {
+    mouseEvent.stopPropagation(); // to prevent collapsing if collapsible
+    setShowingAbsoluteAddresses(!showingAbsoluteAddresses);
```

Comment:
Small update here - when updating a state value that depends on the previous value, it's better to use the callback version:

```ts
setShowingAbsoluteAddresses(oldValue => !oldValue);
```

For this scenario it should work exactly the same but it helps prevents some weird bugs being introduced later since the state value can be out of date sometimes.